File: esb-produto-api/app/model/Produto.scala
package model

import play.api.libs.json._

case class Produto(id: Int, nome: String, descricao: String, foto: String, valor: Float)

object Produto {

  implicit val implicitProdutoWrites = new Writes[Produto] {
    def writes(produto: Produto): JsValue = {
      Json.obj(
        "id" -> produto.id,
        "nome" -> produto.nome,
        "descricao" -> produto.descricao,
        "foto" -> produto.foto,
        "valor" -> produto.valor
      )
    }
  }
}

File: esb-mule/src/main/java/com/github/dsaouda/fiapesb/model/Produto.java
package com.github.dsaouda.fiapesb.model;

public class Produto {
	
	private long id;
	private String nome;
	private String descricao;
	private String foto;
	private float valor;
	
	public long getId() {
		return id;
	}
	
	public void setId(long id) {
		this.id = id;
	}
	
	public String getNome() {
		return nome;
	}
	
	public void setNome(String nome) {
		this.nome = nome;
	}
	
	public String getDescricao() {
		return descricao;
	}
	
	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}
	
	public String getFoto() {
		return foto;
	}
	
	public void setFoto(String foto) {
		this.foto = foto;
	}
	
	public float getValor() {
		return valor;
	}
	
	public void setValor(float valor) {
		this.valor = valor;
	}
	
	@Override
	public String toString() {
		return "Produto [id=" + id + ", nome=" + nome + ", descricao=" + descricao + ", foto=" + foto + ", valor="
				+ valor + "]";
	}
}
